pkg/downloader/manager.go

```diff
@@ -323,6 +325,14 @@ func (m *Manager) downloadAll(deps []*chart.Dependency) error {
 			continue
 		}
 
+		if m.SkipDownloadIfExists {
+			name := filepath.Base(churl)
```

Comment:
```suggestion
			name := dep.Name + "-" + dep.Version + ".tgz"
```

This is the format that matches what I pull from our internal (Google artifact) registry. I don't know what formats could be downloaded from other providers. I'd imagine this needs a format agnostic way of checking the `charts/` directory.

Reply:
Or alternatively checking for the existence of `filepath.Base(churl)` OR `dep.Name + "-" + dep.Version + ".tgz"` should cover the majority of cases and worse case if no match the files will be downloaded.,

Reply:
added support for oci